src/functions-and-arrays.js

```diff
@@ -1,20 +1,46 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers() {
+  if (a > b) {
+    return a 
+   } else if (a < b) {
+    return b
+   } else if (a === b) {
+    return a || b
```

Comment:
This will always return `a`, because `a` is always truthy. You wouldn't need the last else-if-statement, because if both numbers are equal, you just can return `b`.